mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
Looks like Rackspace uses the name `cloudBlockStorage`

```
$ curl -H "Content-Type: application/json" -s -XPOST -d '{"auth":{"RAX-KSKEY:apiKeyCredentials":{"username":"joe.blogs.example.com","apiKey":"111222333"}}}' https://identity.api.rackspacecloud.com/v2.0/tokens | python -m json.tool
...
{
    "access": {
        "serviceCatalog": [
            ...
            {
                "endpoints": [
                    ...
                    {
                        "publicURL": "https://iad.blockstorage.api.rackspacecloud.com/v1/123456",
                        "region": "IAD",
                        "tenantId": "123456"
                    }
                ],
                "name": "cloudBlockStorage",
                "type": "volume"
            },
        ],
        ...
    }
}
```

I'll change this back for now because I don't think the name makes any difference in my use case but perhaps this needs to be configurable or perhaps Mimic could grow a ``--rackspace / --devstack`` compatibility option.